### ISSUE TITLE
Tag input is not supported

### ISSUE THREAD

**Issue by maxkruger** [google-github-actions/deploy-cloudrun#160]
### TL;DR
When using `tag` input (as listed [here](https://github.com/google-github-actions/deploy-cloudrun#inputs)), the error below is experienced

**Expected behavior**
The revision to be tagged with the value supplied

**Observed behavior**
`Warning: Unexpected input(s) 'tag', valid inputs are ['image', 'service', 'region', 'env_vars', 'metadata', 'project_id', 'credentials']
Run google-github-actions/deploy-cloudrun@v0.2.0
  with:
    service: ***
    image: gcr.io/***/***
    credentials: ***
    tag: 0.1.1-rc.2
  env:
    CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
Setting project Id from credentials
Creating a new service...
Service *** has been successfully deployed.
Error: The operation was canceled.`

### Reproduction

**Action YAML**

```yaml
name: Stage application on Google Cloud Run
on:
  push:
    tags:
    - '*.*.*-rc.*'

jobs:
  deploy:
    name: Deploy job
    runs-on: ubuntu-latest
    steps:
    - name: Checkout the repository
      uses: actions/checkout@v1

    - name: Build Docker image
      run: |
        docker build . --tag eu.gcr.io/${{ secrets.GCLOUD_PROJECT }}/${{ secrets.GCLOUD_APP_NAME }}

    - name: Authenticate into Google Cloud Platform
      uses: google-github-actions/setup-gcloud@master
      with:
        service_account_email: ${{ secrets.GCLOUD_EMAIL }}
        service_account_key: ${{ secrets.GCLOUD_AUTH }}

    - name: Configure Docker to use Google Cloud Platform
      run: "gcloud auth configure-docker --quiet"

    - name: Push image to Google Cloud Container Registry
      run: docker push eu.gcr.io/${{ secrets.GCLOUD_PROJECT }}/${{ secrets.GCLOUD_APP_NAME }}

    - id: get_version
      uses: battila7/get-version-action@v2

    - name: Deploy to Cloud Run
      uses: google-github-actions/deploy-cloudrun@v0.2.0
      with:
        service: ${{ secrets.GCLOUD_APP_NAME }}
        image: gcr.io/${{ secrets.GCLOUD_PROJECT }}/${{ secrets.GCLOUD_APP_NAME }}
        region: ${{ secrets.GCLOUD_REGION }}
        credentials: ${{ secrets.GCLOUD_AUTH }}
        tag: ${{ steps.get_version.outputs.version }}

```

**Repository**

**Additional information**

**Comment by averikitsch** [comment]
hi @maxkruger please update to the latest release `google-github-actions/deploy-cloudrun@v0.6.0`.

**Comment by maxkruger** [comment]
Thanks 👍

**Comment by igorkf** [comment]
Hi @averikitsch 
I'm trying to use `env_vars` but seems that Github Actions is ignoring.   
Check the part of `Deploy to Cloud Run`:
```
- name: Deploy to Cloud Run
        id: deploy
        uses: google-github-actions/deploy-cloudrun@0.6.0 # support for env_vars
        with:
          service: ${{ env.SERVICE }}
          image: gcr.io/${{ env.PROJECT_ID }}/${{ env.SERVICE }}:${{  github.sha }}
          region: ${{ env.REGION }}
          env_vars: MODE_TOKEN=${{ secrets.MODE_TOKEN }},MODE_PASS=${{ secrets.MODE_PASS }},DISC_WEBHOOK_DATACHANNEL=${{ secrets.DISC_WEBHOOK_DATACHANNEL }}
```

Am I doing something wrong?  
I would like to use these environment variables in my project.

**Comment by igorkf** [comment]
Hi, @averikitsch 
I tried to run the Github Actions using `google-github-actions/deploy-cloudrun@v0.6.0` for setting environment variable with `env_vars`, but seems that `env_vars` is being ignored.
